governance/program/src/instruction.rs

```diff
@@ -273,40 +314,51 @@ pub enum GovernanceInstruction {
+    ///   3. `[signer]` Signatory account signing off the Proposal Or Proposal
+    ///      owner if the owner hasn't appointed any signatories
+    ///   4. `[]` TokenOwnerRecord for the Proposal owner, required when the
+    ///      owner signs off the Proposal Or `[writable]` SignatoryRecord
+    ///      account, required when non owner sings off the Proposal
     SignOffProposal,
 
-    ///  Uses your voter weight (deposited Community or Council tokens) to cast a vote on a Proposal
-    ///  By doing so you indicate you approve or disapprove of running the Proposal set of transactions
-    ///  If you tip the consensus then the transactions can begin to be run after their hold up time
+    ///  Uses your voter weight (deposited Community or Council tokens) to cast
+    /// a vote on a Proposal  By doing so you indicate you approve or
+    /// disapprove of running the Proposal set of transactions  If you tip
+    /// the consensus then the transactions can begin to be run after their hold
+    /// up time
     ///
     ///   0. `[]` Realm account
     ///   1. `[writable]` Governance account
     ///   2. `[writable]` Proposal account
     ///   3. `[writable]` TokenOwnerRecord of the Proposal owner
-    ///   4. `[writable]` TokenOwnerRecord of the voter. PDA seeds: ['governance',realm, vote_governing_token_mint, governing_token_owner]
-    ///   5. `[signer]` Governance Authority (Token Owner or Governance Delegate)
-    ///   6. `[writable]` Proposal VoteRecord account. PDA seeds: ['governance',proposal,token_owner_record]
-    ///   7. `[]` The Governing Token Mint which is used to cast the vote (vote_governing_token_mint)
-    ///           The voting token mint is the governing_token_mint of the Proposal for Approve, Deny and Abstain votes
-    ///           For Veto vote the voting token mint is the mint of the opposite voting population
-    ///           Council mint to veto Community proposals and Community mint to veto Council proposals
-    ///           Note: In the current version only Council veto is supported
+    ///   4. `[writable]` TokenOwnerRecord of the voter. PDA seeds:
+    ///      ['governance',realm, vote_governing_token_mint,
+    ///      governing_token_owner]
+    ///   5. `[signer]` Governance Authority (Token Owner or Governance
+    ///      Delegate)
+    ///   6. `[writable]` Proposal VoteRecord account. PDA seeds:
+    ///      ['governance',proposal,token_owner_record]
+    ///   7. `[]` The Governing Token Mint which is used to cast the vote
+    ///      (vote_governing_token_mint) The voting token mint is the
+    ///      governing_token_mint of the Proposal for Approve, Deny and Abstain
+    ///      votes For Veto vote the voting token mint is the mint of the
+    ///      opposite voting population Council mint to veto Community proposals
+    ///      and Community mint to veto Council proposals Note: In the current
+    ///      version only Council veto is supported
```

Comment:
Need some periods here
```suggestion
    ///   7. `[]` The Governing Token Mint which is used to cast the vote
    ///      (vote_governing_token_mint). The voting token mint is the
    ///      governing_token_mint of the Proposal for Approve, Deny and Abstain
    ///      votes. For Veto vote the voting token mint is the mint of the
    ///      opposite voting population. Council mint to veto Community proposals
    ///      and Community mint to veto Council proposals.
    ///      Note: In the current
    ///      version only Council veto is supported
```